dynamic-simulation/dynamic-simulation-dsl/src/test/java/com/powsybl/dynamicsimulation/groovy/DummyDynamicModel.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.dynamicsimulation.groovy;
+
+import java.util.Objects;
+
+import com.powsybl.dynamicsimulation.DynamicModel;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class DummyDynamicModel implements DynamicModel {
+
+    private final String id;
+
+    private final String parameterSetId;
+
+    private final String parameterSetFile;
```

Comment:
Why do we have the parameterSetFile here? I was expecting to have only an Id (that will be either the static Id o the dynamic model Id) and a parameter Id. Am I wrong?

Reply:
Done